app/assets/stylesheets/scaffolds.scss

```
@@ -93,3 +93,16 @@ body.dark-theme {
 body.trusted-status-true .trusted-visible-block {
   display: flex !important;
 }
+
+// The .admin-help-button class is included in this file since it
+// is referenced in both the admin and application layout.
+
+.admin-help-button {
+  position: fixed;
+  margin-bottom: env(safe-area-inset-bottom);
+  border-radius: 50px;
```

```suggestion
  margin-bottom: env(safe-area-inset-bottom);
```
I've removed the other's because we've applied them as utility classes.